src/components/SearchResult/SearchResult.tsx

```diff
@@ -80,14 +95,49 @@ export const SearchResult: FC<SearchResultProps> = ({ title, found, append = fal
   }, [favorites]);
 
   const handleDetailedClose = useCallback(() => setDetailedDoc(null), []);
+  const handleForAllTime = useCallback(() => {
+    setKeywordsToShow(keywords);
+    setAreKeywordsShown(true);
+  }, [keywords]);
+  const toggleDatePicker = useCallback(() => setIsPickerOpened(prev => !prev), []);
+  const handleDatePickerOk = useCallback(async (dateRange: DateRange) => {
+    try {
+      setKeywordsToShow(await axios.post<KeywordDistributionData>('/viewing_representation_keywords', {
```

Comment:
I've faced an issue to make this post method
It might be the backend issue though